user-profile-card.js

```diff
@@ -2,14 +2,19 @@ import '@brightspace-ui/core/components/colors/colors.js';
 import '@brightspace-ui/core/components/icons/icon.js';
 import { bodySmallStyles, bodyStandardStyles, heading2Styles, labelStyles } from '@brightspace-ui/core/components/typography/styles.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import {classMap} from 'lit-html/directives/class-map.js';
```

Comment:
Nit: added spaces around the import is convention
```suggestion
import { classMap } from 'lit-html/directives/class-map.js';
```